### PR TITLE
feat: [sc-25032] Refactor NameGuard Landing Page

### PR DESCRIPTION
**Pull request by eduramme**
Story details: https://app.shortcut.com/ps-web3/story/25032

### CODE REVIEW

File: apps/nameguard.io/tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
       }
     ],
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./src/*"],
+      "@components": ["./src/components/"],
+      "@components/*": ["./src/components/*"]
```
Comment (lightwalker-eth) [inline apps/nameguard.io/tsconfig.json:24]: Can you help me understand why we need to include these additional paths? Thanks :smile:

File: apps/nameguard.io/src/components/organisms/ComingSoonSections.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      </>
+    ),
+    sectionDescription:
+      "In the hustle and bustle of life, ENS name renewals can slip through the cracks. Give your community peace of mind (and earn recurring revenue!) with ENS AutoRenew. What’s more, it also helps everyone save on gas fees, intelligently initiating renewal transactions at the most cost-effective moments. ",
+    sectionBackgroundName: "bg-green_background",
+    isTextOnTheLeft: false,
+    badgeText: "Planned",
+    imageSpecifics: {
+      source: autorenew_img.src,
+      tagWidth: 11600,
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/organisms/ComingSoonSections.tsx:64]: Wow seems big. Could you please double check these?

File: apps/nameguard.io/src/components/organisms/ReadySection.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          <div
+            className={cc([
+              "absolute z-0 top-[40%] sm:top-[30%] left-[10%] h-[60%] w-[80%] bg-center bg-no-repeat bg-cover [opacity:0.4]",
+              props.sectionBackgroundName,
+            ])}
+          />
+          <Image
+            className="z-10 w-full h-full max-w-[34rem] xl:w-1/3 xl:h-auto"
+            src={props.imageSpecifics.source}
+            alt="chat image"
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/organisms/ReadySection.tsx:54]: Is this alt text correct?

**Review by lightwalker-eth — COMMENTED** [review]
@eduramme Great to see these updates! I know this PR is still just a draft but I had a quick look over it. Reviewed and shared a few comments with feedback for you :+1:

File: apps/nameguard.io/tsconfig.json (hunk 1 above)
Comment (eduramme) [reply apps/nameguard.io/tsconfig.json]: I deleted those 😄
I wanted to set up an alias for importing components using "@components/...". However, we already have a similar shortcut for the 'src' folder, so changing it from "@/components" to "@components" doesn't really help much.

File: apps/nameguard.io/src/components/organisms/ReadySections.tsx [hunk 4]
```diff
@@ -73,6 +73,7 @@ const readySections: ReadySectionProps[] = [
     ),
     imageSpecifics: {
       source: impersonation_attack_img.src,
+      alt: "Alert from deceptive impersonation attacks image",
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/organisms/ReadySections.tsx:76]:
```suggestion
      alt: "Alert from deceptive impersonation attacks",
```

File: apps/nameguard.io/src/components/organisms/ReadySections.tsx [hunk 5]
```diff
@@ -120,6 +121,7 @@ const readySections: ReadySectionProps[] = [
     ),
     imageSpecifics: {
       source: fake_NFT_filter_img.src,
+      alt: "Filter out fake ENS NFTs image",
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/organisms/ReadySections.tsx:124]:
```suggestion
      alt: "Filter out fake ENS NFTs",
```

File: apps/nameguard.io/src/components/organisms/ReadySections.tsx [hunk 6]
```diff
@@ -144,13 +146,14 @@ const readySections: ReadySectionProps[] = [
...
+const name = "abc123🚀.eth";
+const result = await nameguard.inspectName(name);
 
-    const names = ["abc.eth", "123.eth", "xyz.eth"];
-    const results = await nameguard.bulkInspectNames(names);`,
+const names = ["abc.eth", "123.eth", "xyz.eth"];
+const results = await nameguard.bulkInspectNames(names);`,
     imageSpecifics: {
       source: surface_risks_img.src,
+      alt: "Surface hidden risks or limitations image",
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/components/organisms/ReadySections.tsx:156]:
```suggestion
      alt: "Surface hidden risks or limitations",
```

**Review by lightwalker-eth — APPROVED** [review]
@eduramme great updates :+1: Thank you! Approved. Let's see what @notrab can suggest for fixing the storybook build.

**Review by lightwalker-eth — COMMENTED** [review]
@eduramme Updates look good. Before I approve this can you please resolve the merge conflicts?

Also suggest we resolve https://github.com/namehash/nameguard/pull/294#pullrequestreview-2156115064 and merge it into main too and resolve any merge conflicts in this PR before we merge this PR. What do you think?

**Comment by eduramme** [comment]
> @eduramme Updates look good. Before I approve this can you please resolve the merge conflicts?
> 
> Also suggest we resolve [#294 (review)](https://github.com/namehash/nameguard/pull/294#pullrequestreview-2156115064) and merge it into main too and resolve any merge conflicts in this PR before we merge this PR. What do you think?

Sounds good to me

**Review by lightwalker-eth — APPROVED** [review]
@eduramme this PR is approved but I just merged PR 294 into main which means that this PR (295) has more merge conflicts to resolve. Could you please resolve these and then directly merge this into main without my review? Thanks!

**Pull request merged**